app/src/main/java/org/xbmc/kore/utils/NetUtils.java

```diff
@@ -194,23 +194,6 @@ public static boolean sendWolMagicPacket(String macAddress, String hostAddress,
         return true;
     }
 
-    private static byte[] getMacBytes(String macStr) throws IllegalArgumentException {
-        byte[] bytes = new byte[6];
-        String[] hex = macStr.split("(\\:|\\-)");
-        if (hex.length != 6) {
-            throw new IllegalArgumentException("Invalid MAC address.");
-        }
-        try {
-            for (int i = 0; i < 6; i++) {
-                bytes[i] = (byte) Integer.parseInt(hex[i], 16);
-            }
-        }
-        catch (NumberFormatException e) {
-            throw new IllegalArgumentException("Invalid hex digit in MAC address.");
-        }
-        return bytes;
-    }
-
```

Comment:
This one can be useful in the future for some reason that i can't think of right now... I would keep it.